src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const initialState = [];
+
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Great job defining the `CHECK_STATUS` action type as requested by the previous code reviewer, but according to the project[ requirements.](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md) it is supposed to follow the [ducks](https://github.com/erikras/ducks-modular-redux) pattern, please let's 
 fix that. It should be something like so: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`.

Reply:
Done!